File: Java/Program/Product.java
public class Product {
    private int idProduct;
    private String name;
    private String brand;
    private String price;

    public Product(){
    }

    public Product(int idProduct, String name, String brand, String price){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public int getID(){
        return this.idProduct;
    }

    public void setID(int idProduct){
        this.idProduct = idProduct;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getBrand(){
        return this.brand;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }
    public String getPrice(){
        return this.price;
    }

    public void setPrice(String price){
        this.price = price;
    }
}

File: Python/Program/Product.py
class Product:
    __idProduct = 0
    __name = ""
    __brand = ""
    __price = ""

    def __init__(self, idProduct = 0, name = "", brand = "", price = ""):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.__price = price

    def getID(self):
        return self.__idProduct
    
    def setID(self, idProduct):
        self.__idProduct = idProduct

    def getName(self):
        return self.__name
    
    def setName(self, name):
        self.__name = name

    def getBrand(self):
        return self.__brand

    def setBrand(self, brand):
        self.__brand = brand

    def getPrice(self):
        return self.__price
    
    def setPrice(self, price):
        self.__price = price
